core/src/main/java/org/sonarsource/sonarlint/core/analyzer/issue/DefaultClientIssue.java

```diff
@@ -85,6 +88,10 @@ public ClientInputFile getInputFile() {
     return flows;
   }
 
+  public List<QuickFix> quickFixes() {
```

Comment:
As discussed we can also add the method in the `Issue` interface and mark the override here